service/src/main/kotlin/xyz/lebkuchenfm/Application.kt

```diff
@@ -150,20 +158,14 @@ fun Application.module() {
     }
 
     routing {
-        // TODO: This route should be secured using auth-session and auth-bearer when the whole auth flow is done.
-        route("/api") {
-            authRouting()
-            xSoundsRouting(xSoundsService)
-            songsRouting(songsService)
-            commandsRouting(commandExecutorService)
-            eventStreamRouting(eventStream, playerStateSynchronizer)
-        }
-
-        // TODO: Remove me.
         authenticate("auth-session") {
             authenticate("auth-bearer") {
-                get("/auth-test") {
-                    call.respondText { "You are authenticated!" }
+                route("/api") {
+                    authRouting()
```

Comment:
im guessing that auth routing is not yet finished and we have to prepare desired login screen?

Reply:
Everything about routing is already done, including login screen which lives inside client-beta package.